prowler/providers/gcp/exceptions/exceptions.py

```diff
@@ -85,3 +93,17 @@ def __init__(self, file=None, original_exception=None, message=None):
         super().__init__(
             1930, file=file, original_exception=original_exception, message=message
         )
+
+
+class GCPLoadCredentialsFromDictError(GCPBaseException):
```

Comment:
Can you create a `GCPCredentialsError` class for this two?

Reply:
Done!